x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/ClusterPrivilegeResolver.java

```diff
@@ -153,7 +154,8 @@ public class ClusterPrivilegeResolver {
 
     private static final Set<String> CROSS_CLUSTER_ACCESS_PATTERN = Set.of(
         RemoteClusterService.REMOTE_CLUSTER_HANDSHAKE_ACTION_NAME,
-        RemoteClusterNodesAction.NAME
+        RemoteClusterNodesAction.NAME,
+        XPackInfoAction.NAME
```

Comment:
Yes. With the new model, it is no longer necessary for individual feature to check the license. However, I think allowing it for now is the better choice because:
1. Licensing for different feature is rather independant. Hence there is no inherent relationship between RCS 2 and transform and one cannot assume the other.
2. Dropping the license check requires fair amount of code changes on the transform and ML side. The cost benefit ration just does not tick.
3. If we are giong to allow RCS 2 to work partially with basic license, the individual license checking is again necessary (this echoes to the 1st point that license decisions are independent between features).